### PR TITLE
Add a delays page for alert analysis

### PR DESCRIPTION
**Pull request by devinmatte**
## Motivation

#986

Keep track of what is keeping the MBTA from reliably delivering service on each line

## Changes

Screenshot 2024-09-02 at 9 18 40 PM

Screenshot 2024-09-03 at 9 10 04 PM

## Testing Instructions

Requires data from https://github.com/transitmatters/data-ingestion/pull/104 and https://github.com/transitmatters/data-ingestion/pull/113 before being merged

### CODE REVIEW

File: modules/reliability/charts/DelayBreakdownGraph.tsx [hunk 1]
```diff
@@ -0,0 +1,305 @@
...
+  const ref = useRef();
+  const isMobile = !useBreakpoint('md');
+  const labels = data.map((point) => point.date);
+
+  const lineColor = LINE_COLORS[line ?? 'default'];
+
+  return (
+    <ChartBorder>
+      <div className={classNames('h-72', 'flex w-full flex-row')}>
+        <Line
```
Comment (idreyn) [inline modules/reliability/charts/DelayBreakdownGraph.tsx:42]: I'm curious if `TimeSeriesChart.tsx` would work for your purposes. I've been hoping we could consolidate some of the chart.js boilerplate into this component, and I had success using it for some of the newer service/ridership graphs. Not a big deal either way (but if you try it and find that it's not working somehow, I'd love to know why!)

**Comment by rudiejd** [comment]
This closes https://github.com/transitmatters/t-performance-dash/issues/986, right?

**Comment by devinmatte** [comment]
Yep @rudiejd, that's the plan

**Review by skaplan-dev — APPROVED** [review]
I do think this PR is a little too big and could benefit from separating the server changes from the FE changes. But LGTM 🚢

**Comment by devinmatte** [comment]
Backend changes in https://github.com/transitmatters/t-performance-dash/pull/1011 to shrink this PR

**Pull request merged**